README.rst

```diff
@@ -327,6 +327,11 @@ be protected with authentication or not.
 Extending django-x509
 ---------------------
 
+The django app ``tests/openwisp2/sample_x509/`` adds some changes on
+top of the ``django-x509`` module with the sole purpose of testing the
+module's extensibility. It can be used as a sample for extending
+``django-x509`` functionality in your own application.
+
```

Comment:
I think we need to add a mention of how to reuse the base tests.

Don't forget the premise I mentioned recently in the mailing list (see "premise" here: https://github.com/openwisp/openwisp-firmware-upgrader#extending-openwisp-firmware-upgrader)